apps/quark/quark.yml

```diff
@@ -0,0 +1,20 @@
+name: Quark
+description: 'Software sketchbook for your node projects.'
+website: 'https://quarkjs.io'
+repository: 'https://github.com/Nishkalkashyap/Quark-electron'
+category: 'Developer Tools'
+keywords:
+    - electron
```

Comment:
Every app in repo is an Electron app. I'm nto sure that these two lines will help much :slightly_smiling_face: 

```suggestion
```